src/FESpaces.jl

```diff
@@ -318,6 +318,20 @@ function FESpaces.TrialFESpace(fun,f::DistributedSingleFieldFESpace)
   DistributedSingleFieldFESpace(spaces,f.gids,f.vector_type)
 end
 
+function FESpaces.TrialFESpace!(f::DistributedSingleFieldFESpace,fun)
```

Comment:
Add some test that call these new functions

Reply:
Done in 910f48af510d2b36a7cb10a1033b95f433a0b77a